test/TestClass.cpp

```diff
@@ -213,13 +213,17 @@ unittest(blink_high) {
 unittest(cursor_high) {
   // create lcd object
   LiquidCrystal_Test lcd(rs, enable, d4, d5, d6, d7);
-  // TODO
-}
 
-unittest(noCursor_high) {
-  // create lcd object
-  LiquidCrystal_Test lcd(rs, enable, d4, d5, d6, d7);
-  // TODO
+  // assert startup no cursor is set
+  assertEqual(false, lcd.isCursor());
```

Comment:
You could also use `assertFalse(lcd.isCursor());`

Reply:
Thank you, that will make the code look more streamlined.